src/scripts/cfo.zig

```diff
@@ -320,15 +465,19 @@ const SeedRecord = struct {
     commit_timestamp: u64, // compared in inverse order
     commit_sha: [40]u8,
     fuzzer: Fuzzer,
-    ok: bool,
-    seed_timestamp_start: u64,
-    seed_timestamp_end: u64,
-    seed: u64,
-    command: []const u8, // excluded from comparison
+    ok: bool = false,
+    seed_timestamp_start: u64 = 0,
+    seed_timestamp_end: u64 = 0,
+    seed: u64 = 0,
+    // The following fields are excluded from comparison:
+    command: []const u8 = "",
+    // TODO: when all seeds in the db have a branch specified, remove this default.
+    branch: []const u8 = "",
```

Comment:
Is `branch` meant to be the name of the branch (eg, `main`) or a link to it on GitHub (`.branch = "https://github.com/tigerbeetle/tigerbeetle",`)?

Reply:
The URL. It is used for:

* UI-level grouping of failing seeds (so that seeds from main don't "shadow" seeds from PRs when displaying)
* UI links to the code

so, anything that's sufficiently unique and sufficiently clickable works.